Test/unit/Controllers/order.test.js

```diff
@@ -0,0 +1,98 @@
+const { createOrder } = require('../../../Services/user/Api/Controllers/order');
+const OrderController = require('../../../Services/user/Api/Controllers/order');
+const chai = require('chai');
+const sinon = require('sinon');
+const sandBox = sinon.createSandbox();
+const chaiAsPromised = require("chai-as-promised");
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+const assert = chai.assert;
+const { Order, Product } = require('../../../Services/user/database/models/');
+const mocks = require('../../Mocks/controllers/order.controller');
+chai.should();
+
+describe('Test to OrderController', async () => {
+
+  beforeEach(() => {
+    sandBox.restore();
+  });
+
+
+  it('Should create a new order', async () => {
+    const callback = sandBox.spy();
+    sandBox.stub(Order, 'create').resolves(mocks.orderCreate);
+    return OrderController.createOrder(mocks.orderCreateCall, callback)
+    .should.be.fulfilled
+    .then(() => {
+      assert(callback.calledOnce);
+      expect(callback.args[0][0]).to.be.a('null');
+      expect(callback.args[0][1]).to.be.a('number');
```

Comment:
```javascript
callback.calledOnce.should.be.true;
callback.args[0][0].to.be.null;
callback.args[0][1].to.be.a('number');
```

Reply:
`callback.args[0][0].should.be.null;` => `callback.args[0].should.be.eql(params);`

Reply:
Done all the code